src/pages/scientificServices/pipelines/views/modals/ViewOutputsModal.tsx

```diff
@@ -65,7 +113,21 @@ export const ViewOutputsModal = ({ jobId, onDismiss }: OutputsModalProps): React
                       backgroundColor: '#f5f5f5',
                     }}
                   >
-                    <div style={{ fontFamily: 'monospace', wordBreak: 'break-all' }}>{key}</div>
+                    <div style={{ flex: 1 }}>
+                      <div style={{ fontFamily: 'monospace', wordBreak: 'break-all', marginBottom: '0.25rem' }}>
+                        {key}
+                      </div>
+                      <div style={{ fontSize: '0.875rem', color: '#666' }}>
+                        {loadingFileSizes[key] ? (
+                          <span style={{ display: 'flex', alignItems: 'center', gap: '0.25rem' }}>
+                            <Spinner size={12} />
+                            Loading size...
+                          </span>
+                        ) : (
+                          fileSizes[key] || 'Unknown size'
```

Comment:
following up on Morgan's suggestion in the Mixpanel PR, I do intend to also capture this info in the mixpanel "downloadOutput" event, but the mixpanel PR hasn't landed on `dev` yet so i'll add that in a very minor follow-up PR